src/main/scala/io/confluent/examples/streams/algebird/CMSStore.scala

```diff
@@ -256,7 +260,7 @@ class CMSStore[T: CMSHasher](override val name: String,
     */
   override def flush() {
     if (loggingEnabled) {
-      changeLogger.logChange(changelogKey, cms)
+      changeLogger.logChange(changelogKey, cms, timestampOfLastStateStoreUpdate)
```

Comment:
Hmm.. instead of passing the timestamp should we just get it from context as did in `StoreChangeLogger`?

Reply:
We cannot access the context here but get an exception. That's the reason for the change. Note, `flush()` is called on commit, ie, not during processing.

I am not sure why we were able to call `context.timestamp` in the old version -- some PR must have fixed this bug (not sure which one)

Reply:
yep - i don't think this was previously called before i made the changes for KIP-182